### PR TITLE
Personal portfolio deploy

### PR DESCRIPTION
**Pull request by mprotic123**
This pull request is created in order to review the project "Portfolio: application deploy" from Microverse.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+## Built With
+
+- HTML
+- CSS (using Flex and Grid)
+
+## Getting Started
+
+**To clone this project**
+**Run command: $git clone [name of the repo]**
+**or press the green Code button in the upper right corner and choose to download from the Download ZIP link.**
```
Comment (vichuge) [inline README.md]: - [ ] To have a more professional readme, I invite you to write this as bullets like other sections on this readme file. 🐶 

- **To clone this project**
- **Run command: $git clone [name of the repo]**
- **or press the green Code button in the upper right corner and choose to download from the Download ZIP link.**
```
- **To clone this project**
- **Run command: $git clone [name of the repo]**
- **or press the green Code button in the upper right corner and choose to download from the Download ZIP link.**
```

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] It looks like you're having troubles with deployment, when I open the page, exist some elements taking more space on the right side, I add border red to all elements on the project to check it out. I kindly invite you to work on it.
image
- Hint: I think is something with URL, if you observe you're using the same URL default for all your deployments as the project name, maybe this could have troubles on the GH pages (`https://mprotic123.github.io/`). When I click the project `Welcome button` the page fix a little bit, I think all these problems are because you use the same URL address as the project name. 
image

- Look at this example, a GitHub deployment should have your GH nick, then `.github.io/` and followed to the project name
image



- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
> # Required Changes ♻️ ⚠️ 🚦
> Hi dear student,
> 
> Good job so far!
> You still need to work on some issues to go to the next project, but you are almost there!
> 
> mario_ladders
> 
> ## Good points 👍
> You used proper GitHub Flow ✔️
> Your linter checks are all passing ✔️
> Your README and PR message are descriptive ✔️
> 
> ## Aspects to improve ♻️
> * [ ]  It looks like you're having troubles with deployment, when I open the page, exist some elements taking more space on the right side, I add border red to all elements on the project to check it out. I kindly invite you to work on it.
>   image
> * Hint: I think is something with URL, if you observe you're using the same URL default for all your deployments as the project name, maybe this could have troubles on the GH pages (`https://mprotic123.github.io/`). When I click the project `Welcome button` the page fix a little bit, I think all these problems are because you use the same URL address as the project name.
>   image
> * Look at this example, a GitHub deployment should have your GH nick, then `.github.io/` and followed to the project name
>   image
> * [ ]  Check the comments under the review.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@mprotic123, I hope I'm not missing anything, you still need to work on the changes mentioned by the previous reviewer.

You still need to work on the element alignment(too much space on the right side), and also your GH pages.

Hint: kindly watch the[ GH pages tutorial](https://www.youtube.com/watch?v=QyFcl_Fba-k)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @mprotic123  👏 , 

 ## Required Changes ♻️
 
_Check the comments above the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**